app/authentication_views.py

```diff
@@ -20,48 +25,78 @@ def register():
             return "at least 1 capital letter, 1 small letter, 1 number, length 8 - 20"
 
         if not validation.validate_email(email):
+            logger.error(
+                f"Handling '/register' route, registering new user {username} failed: invalid email format")
             return "wrong email format"
 
-        if UsersRepository.create(username, email, crypto.hash_password(password), birthday):
-            return "success"
+        if not UsersRepository.create(username, email, crypto.hash_password(password), birthday):
+            return "User has been not created"
 
-        return "Something went wrong"
+        logger.info(
+            f"Handling '/register' route, registering new user {username} is finished successfully")
+        return "success"
 
     else:
+        logger.info(
+            "Handling '/register' route, user is not authenticated and from not submitted, rendering register.html")
         return render_template("forms/register.html")
 
 
 @app.route("/login", methods=["POST", "GET"])
 def login():
 
+    logger.info("Handling '/login' route")
     if "user" in session:
+        logger.info(
+            "Handling '/login' route,  user is authenticated, redirecting to route '/home'")
         return redirect(url_for("home"))
 
-    elif request.method == "POST":
-        username = request.form['username']
-        password = request.form['password'] 
+    if request.method == "POST":
 
+        username = request.form['username']
+        password = request.form['password']
         user = UsersRepository.get_by_name(username)
+        logger.info(
+            f"Handling '/login' route, login form is submitted. Form details username field: {username}")
+
+        # TODO change responses
+        if not user:
+            logger.error(
+                f"Handling '/login' route, authenticating user {username} failed: Invalid username")
+            return "invalid username"
+        if not user.password == crypto.hash_password(password):
+            logger.error(
+                f"Handling '/login' route, authenticating user {username} failed: Invalid password")
+            return "invalid password"
+
+        session["user"] = user.to_json()
+        logger.info(
+            f"Handling '/login' route, authenticating user {username} finished successfully")
+        logger.info(
+            "Handling '/login' route, user is authenticated. Redirecting to route 'home'")
+        return redirect(url_for("home"))
 
-        if user:
-
-            if user.password == crypto.hash_password(password):
-
-                session["user"] = user.to_json()
-
-                return redirect(url_for("home"))
-
-        return "something went wrong"
-
-    else:
-        return render_template("forms/login.html")
+    logger.info(
+        "Handling '/login' route, user is not authenticated. Rendering login.html")
```

Comment:
I saw you decided to use ', rendering'
use ', redirecting' too. Keep same format everywhere

Reply:
I was looking for it before committing but of course, I missed them.